js/window.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+// (0, eval) is indirect eval.
+// See the links below for details:
+// - https://stackoverflow.com/a/14120023
+// - https://tc39.github.io/ecma262/#sec-performeval (spec)
+export const window = (0, eval)("this");
+// TODO: The above should be replaced with globalThis
```

Comment:
`globalThis` is already in the runtime... We should be able to do this now, but when I tried it before, there was some sort of error in the snapshotting, I don't know if the snapshot is being done with the right config or some other issue.